assets/js/modules/analytics-4/utils/report-rows-with-set-values.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Analytics 4 filter rows having (not set) values.
```

Comment:
Let's tweak the description to be a bit clearer:

```suggestion
 * Utility to filter out Analytics report rows which don't have a value set.
 ```